src/app/components/modules/gpa-calculator/gpa-calculator.component.html

```diff
@@ -0,0 +1,42 @@
+<div #moduleContainer class="module-container">
+  <!-- <div *ngFor="let class of displayClassesArray">
+    <h1>{{class}}</h1>
+  </div> -->
+  <table class="table">
+    <thead class="thead-light">
+      <tr>
+        <th class="class-name">Class</th>
+        <th class="letter-grade">Grade</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr *ngFor="let class of displayClassesArray; let i = index">
+        <td class="class-name">{{ class }}</td>
+        <td class="letter-grade">
+          <select
+            class="form-control"
+            name="grade"
+            [(ngModel)]="inputGrades[i]"
+          >
+            <option *ngFor="let grade of letterGradesArray" [value]="grade">{{
+              grade
+            }}</option>
+          </select>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+  <div class="button-wrapper">
+    <a
+      type="button"
+      class="btn btn-success"
+      (click)="calculateGpa()"
+      tooltip="We never save your grades"
```

Comment:
This is good, but personally I think it needs to be more important in the module. Maybe not its own paragraph, but somewhere other than a tooltip.

Reply:
how about a `<p>` tag that appears after calculation?